pkg/pgmodel/migrate.go

```diff
@@ -225,7 +241,17 @@ func ensureVersionTable(db *pgxpool.Pool) error {
 
 func getDBVersion(db *pgxpool.Pool) (semver.Version, error) {
```

Comment:
```suggestion
func getSchemaVersion(db *pgxpool.Pool) (semver.Version, error) {
```

Reply:
I think this change should happen in the PR that adds timescale-version checking